src/main/kotlin/dev/avt/api/admin/absence/AdminAbsenceRouting.kt

```diff
@@ -1,14 +1,68 @@
 package dev.avt.api.admin.absence
 
+import dev.avt.api.admin.absence.RequestedHour.Companion.toRequestedHour
 import dev.avt.api.admin.absence.approveHours.approveHoursRoute
 import dev.avt.api.admin.absence.denyHours.denyHoursRoute
 import dev.avt.api.admin.absence.readRequestedHours.readRequestedHours
 import dev.avt.api.admin.absence.unapproveHours.unapproveHoursRoute
+import dev.avt.database.*
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.auth.*
+import io.ktor.server.response.*
 import io.ktor.server.routing.*
+import kotlinx.serialization.Serializable
+import org.jetbrains.exposed.sql.transactions.transaction
 
 fun Routing.adminAbsenceRouting() {
     approveHoursRoute()
     denyHoursRoute()
     readRequestedHours()
     unapproveHoursRoute()
+
+    route("/api/admin/hours") {
```

Comment:
Admin absence routing is not meant for defining routes like that.
Please create a separate file.

Reply:
I know, I was lazy and waiting for you to do the actual api so you can move it

it's why it's a merge request into admin, not into main